src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -895,6 +973,130 @@ protected ClassLoader configureClassLoader(ClassLoader loader) {
         return context;
     }
 
+    /**
```

Comment:
Not new code, so please only review the _diff_ of these methods with the "2" versions above (left as an exercise for the reader).